process.sh

```diff
@@ -53,24 +53,28 @@ date
 echo '** Exporting KB to OWL **'
 curl -i -X POST ${KBserver}/db/data/transaction/commit -u ${KBuser}:${KBpassword} -H 'Content-Type: application/json' -d '{"statements": [{"statement": "MATCH (c) REMOVE c.label_rdfs RETURN c"}]}' >> ${VFB_DEBUG_DIR}/neo4j_remove_rdfs_label.txt
 curl -i -X POST ${KBserver}/db/data/transaction/commit -u ${KBuser}:${KBpassword} -H 'Content-Type: application/json' -d '{"statements": [{"statement": "MATCH (p) WHERE EXISTS(p.label) SET p.label_rdfs=[] + p.label"}]}' >> ${VFB_DEBUG_DIR}/neo4j_change_label_to_rdfs.txt
+curl -i -X POST ${KBserver}/db/data/transaction/commit -u ${KBuser}:${KBpassword} -H 'Content-Type: application/json' -d '{"statements": [{"statement": "MATCH (n:Entity) WHERE exists(n.block) DETACH DELETE n"}]}' >> ${VFB_DEBUG_DIR}/neo4j_change_label_to_rdfs.txt
+curl -i -X POST ${KBserver}/db/data/transaction/commit -u ${KBuser}:${KBpassword} -H 'Content-Type: application/json' -d '{"statements": [{"statement": "MATCH ()-[r]-() WHERE exists(r.block) DELETE r"}]}' >> ${VFB_DEBUG_DIR}/neo4j_change_label_to_rdfs.txt
```

Comment:
We might want to consider this (although happy to make this a separate ticket for Robbie & myself to consider):

```cql
MATCH (n:Entity) WHERE apoc.convert.toBoolean(ds.block[0]) DETACH DELETE n
MATCH ()-[r]->() WHERE apoc.convert.toBoolean(ds.block[0]) DELETE r
```
This allows for block to explicitly be set to [False]
The absence of a key => False
['Some string'] => True  (unless this is the string 'false') # This allows comments about obsoletion to be stored here and case as True - something Robbie has requested.

Weakness:  pipeline is brittle to value not being a list, although this is currently enforced by kb2kb.

Reply:
I'm happy we shouldn't have a block parameter unless we actually want to block as no reason to encourage block=false being there for no reason.